pkgs/development/tools/avro-tools/default.nix

```diff
@@ -0,0 +1,33 @@
+{ stdenv, fetchurl, makeWrapper, jre, lib }:
+
+stdenv.mkDerivation rec {
+  version = "1.9.0";
+  name = "avro-tools-${version}";
+
+  src = fetchurl {
+    url = "https://repo1.maven.org/maven2/org/apache/avro/avro-tools/${version}/avro-tools-${version}.jar";
+    sha256 = "164mcz7ljd2ikwsq9ba98galcjar4g4n6ag7kkh466nwrpbmd2zi";
+  };
+
+  phases = "installPhase"; # so nix doesn't attempt to decompress
+
+  buildInputs = [ jre ];
+  nativeBuildInputs = [ makeWrapper ];
+  sourceRoot = ".";
+
+  installPhase = ''
+    mkdir -p $out/{bin,share/java}
+    mv $src avro-tools-${version}.jar
+    cp avro-tools-${version}.jar $out/share/java
```

Comment:
I'm thinking if `$out/libexec/avro-tools` woulbe be better directory

Reply:
I don't mind either way.

Reply:
Since I am not sure either, and since there will be no colision, I'm merging this

Reply:
I'd agree that `$out/libexec/avro-tools` would be appropriate.

Reply:
I changed it to `$out/libexec/avro-tools `